src/network/localconnector.cpp

```diff
@@ -0,0 +1,380 @@
+        Qt::UniqueConnection);
+    m_internalBroadcastTimer->start(g_defaultBroadcastCycleMs);
+}
+// ==========================================================================================
+// ==========================================================================================
+// ==========================================================================================
+//                                                       START BROADCASTING SERVER ATTRIBUTES
+void DefaultLocalConnector::startClearingOutdatedAttributes()
+{
+    // Broadcast server attributes each g_defaultBroadcastCycleMs
+    // milliseconds using UDP service
+    m_internalClearOutdatedTimer = std::make_unique<QTimer>();
+    connect(
+        m_internalClearOutdatedTimer.get(), &QTimer::timeout,
+        this,                               &DefaultLocalConnector::clearOutdatedAttributesOnTimerTick,
+        Qt::UniqueConnection);
+    m_internalClearOutdatedTimer->start(g_defaultBroadcastCycleMs);
+}
+// ==========================================================================================
+// ==========================================================================================
+// ==========================================================================================
+//                                                  BROADCAST DATAGRAM WITH SERVER ATTRIBUTES
+void DefaultLocalConnector::shareAttributesOnTimerTick()
+{
+    // Push attributes to the Json bearer string & broadcast it
+    ServerData launchedTcpServerAttrib = m_tcpService->getServerAttributes();
+    QString serverData(launchedTcpServerAttrib.toJsonQString());
+    m_udpService->broadcastDatagram(serverData);
+}
+
+// ==========================================================================================
+// ==========================================================================================
+// ==========================================================================================
+//                                                           CLEAR OUTDATED SERVER ATTRIBUTES
+void DefaultLocalConnector::clearOutdatedAttributesOnTimerTick()
+{
+    /*
+    SizeType nowMomentMs = QDateTime::currentMSecsSinceEpoch();
+    if(m_discoveredTcpServersAttrib.empty())
+    {
+        return;
+    }
+    SizeType oldSize = m_discoveredTcpServersAttrib.size();
+    m_discoveredTcpServersAttrib.erase(
+                std::remove_if(m_discoveredTcpServersAttrib.begin(),
+                               m_discoveredTcpServersAttrib.end(),
+                               [nowMomentMs] (ServerData serverData)
+                               {
+                                   return (nowMomentMs - serverData.m_creationMomentMs) >
+                                          g_defaultOutdatingCycleMs;
+                               }));
+    if (oldSize != m_discoveredTcpServersAttrib.size())
+    {
+        emit onlineUsersUpdated(getOnlineUsers());
+    }
+    */
+}
```

Comment:
Please remove commented code before merging

Reply:
Commented code is already removed.